alerts/alerts.libsonnet

```diff
@@ -35,16 +35,18 @@
           },
           {
             alert: 'BlackboxSslCertificateWillExpireSoon',
-            // Cert-manager defaults to 3 week renewal time
             expr: |||
-              probe_ssl_earliest_cert_expiry{%(blackboxExporterSelector)s} - time() < 21 * 24 * 3600
+              probe_ssl_earliest_cert_expiry{%(blackboxExporterSelector)s} - time() < %(probeSslExpireDaysThreshold)s * 24 * 3600
```

Comment:
Update the tests, if you are unfamiliar with the promtool test framework I can amend your PR.

Thanks for the contribution!